tests/winch.rs

```diff
@@ -36,7 +36,7 @@ async fn test_winch_async() {
     let pts = pty.pts().unwrap();
     pty.resize(pty_process::Size::new(24, 80)).unwrap();
     let mut child = pty_process::Command::new("perl")
-        .args(&[
+        .args([
```

Comment:
Apparently, we had a lint which was not being caught because of running clippy without --all-features.